create-snowpack-app/app-template-react-typescript/package.json

```diff
@@ -20,10 +20,16 @@
     "react-dom": "^16.13.1"
   },
   "devDependencies": {
+    "@babel/plugin-syntax-import-meta": "^7.10.4",
+    "@babel/preset-react": "^7.10.4",
+    "@babel/preset-typescript": "^7.10.4",
     "@snowpack/app-scripts-react": "^1.11.0",
```

Comment:
Remove?

Reply:
Oh I just caught the “everything but testing” bit in the description. Makes sense.